easyInterface/Diffraction/DataClasses/DataObj/Experiment.py

```diff
@@ -254,16 +282,20 @@ def default(cls, polarised: bool = False):
         x = []
         y_obs = []
         sy_obs = []
+        y_obs_diff = None
+        sy_obs_diff = None
         y_obs_up = None
         sy_obs_up = None
         y_obs_down = None
         sy_obs_down = None
         if polarised:
```

Comment:
We should consider to make a enum with type of experiment, instead of using this boolean `polarised`.